src/main/java/uk/gov/justice/hmpps/prison/service/transfer/PrisonTransferService.kt

```diff
@@ -96,25 +97,27 @@ class PrisonTransferService(
       livingUnitMv = null // TODO - is this required if different prison
       assignedLivingUnit = reception
       location = toAgency
-      externalMovementService.updateMovementsForCourtTransferToDifferentPrison(
```

Comment:
all moved and called within notification lambda.
This is because we have to look up the team prior to updating the movement else the "flush" might trigger an update on the booking record, which might remove the team since some teams are automatically removed from the  offender by a NOMIS trigger.


So sequence of calls it

1) Get the assigned team for the offender
2) Update movements and flush
3) Send notification to the offender previous team